src/patternfly/components/ClipboardCopy/examples/ClipboardCopy.md

```diff
@@ -10,17 +10,19 @@ cssPrefix: pf-c-clipboard-copy
   {{#> clipboard-copy-group}}
     {{#> form-control controlType="input" input="true" form-control--attribute=(concat 'type="text" value="This is editable" id="text-input-' clipboard-copy--id '" aria-label="Copyable input"')}}
```

Comment:
{{#> form-control controlType="input" input="true" form-control--attribute=(concat 'type="text" value="This is editable" id="' clipboard-copy--id '-text-input" aria-label="Copyable input"')}}

{{#> button button--modifier="pf-m-control" button--attribute=(concat 'aria-label="Copy to clipboard" id="' clipboard-copy--id '-copy-button'" aria-labelledby="' clipboard-copy--id '-text-input"')}}